mlflow/langchain/langchain_tracer.py

```diff
@@ -29,9 +29,24 @@ class MlflowLangchainTracer(BaseCallbackHandler, metaclass=ExceptionSafeAbstract
     Callback for auto-logging traces.
     We need to inherit ExceptionSafeAbstractClass to avoid invalid new
     input arguments added to original function call.
+    Args:
+        parent_span: Optional parent span for the trace. If provided, the trace
+                     will be created as child of this span. Otherwise, a single trace
```

Comment:
```suggestion
        parent_span: Optional parent span for the trace. If provided, the trace
            will be created as child of this span. Otherwise, a single trace
```